### ISSUE TITLE
Color picker shortcut set the background color

### ISSUE THREAD

**Issue by heitor57** [excalidraw/excalidraw#6922]
Hi, first of all, thank you for all the amazing work you've put into this tool! I've been loving using it.

The issue I'm encountering is with the color picker. When I use any of its shortcuts (e.g., "i", "shift+s", or "shift+g"), it consistently changes the background color instead of the stroke color. This poses a significant problem for me when utilizing the Draw tool, as I wish to swiftly and naturally switch between unique colors.

I believe this could be fixed by introducing distinct shortcuts for each color picker (Stroke and Background) or including a MRU policy in its usage. While I've managed to do color changes using shortcuts ("s+i"), these keys are located quite far apart on the keyboard. This can be challenging, especially considering I use one hand for shortcuts in my setup. If "s+s" were used to open the Draw color picker and "g+g" were employed to open the Background color picker, it would be also an improvement.

**Comment by ad1992** [comment]
@heitor57 to confirm you mean when the selected shape actions panel is open and you want to change stroke color of some shape, using the shortcuts (i, g, s) triggers change in background color instead right ?

**Comment by heitor57** [comment]
@ad1992 Yes, you've comprehended most of it accurately. Although there is indeed a shortcut to change the stroke color, activated by pressing "s+i," this shortcut isn't very ergonomic. It disrupts the smooth flow of drawing and writing when using the Draw tool, as the keys are quite far apart on the keyboard. I'm of the opinion that there should be a simpler shortcut available for altering the stroke color. It would be ideal if this shortcut could be as convenient as changing the background color, which is triggered by just pressing "i".
